heat/fft/tests/test_fft.py

```diff
@@ -0,0 +1,235 @@
+import numpy as np
+import torch
+
+import heat as ht
+from heat.core.tests.test_suites.basic_test import TestCase
+
+
+class TestFFT(TestCase):
+    def test_fft_ifft(self):
+        # 1D non-distributed
+        x = ht.random.randn(6, dtype=ht.float64)
+        y = ht.fft.fft(x)
+        np_y = np.fft.fft(x.numpy())
+        self.assertIsInstance(y, ht.DNDarray)
+        self.assertEqual(y.shape, x.shape)
+        self.assert_array_equal(y, np_y)
+        backwards = ht.fft.ifft(y)
+        self.assertTrue(ht.allclose(backwards, x))
+
+        # 1D distributed
+        x = ht.random.randn(6, split=0)
+        n = 8
+        y = ht.fft.fft(x, n=n)
+        np_y = np.fft.fft(x.numpy(), n=n)
+        self.assertIsInstance(y, ht.DNDarray)
+        self.assertEqual(y.shape, np_y.shape)
+        self.assertTrue(y.split == 0)
+        self.assert_array_equal(y, np_y)
+
+        # n-D distributed
+        x = ht.random.randn(10, 8, 6, dtype=ht.float64, split=0)
+        # FFT along last axis
+        n = 5
+        y = ht.fft.fft(x, n=n)
+        np_y = np.fft.fft(x.numpy(), n=n)
+        self.assertIsInstance(y, ht.DNDarray)
+        self.assertEqual(y.shape, np_y.shape)
+        self.assertTrue(y.split == 0)
+        self.assert_array_equal(y, np_y)
+
+        # on GPU, test only on less than 4 processes
```

Comment:
Is there a reason for not testing on 8 GPUs?

Reply:
I found what caused the problem on more than 3 GPUs and removed this constraint.